jib-core/src/test/java/com/google/cloud/tools/jib/registry/RegistryErrorExceptionBuilderTest.java

```diff
@@ -33,19 +33,19 @@
   @Test
   public void testAddErrorEntry() {
     RegistryErrorExceptionBuilder builder =
-        new RegistryErrorExceptionBuilder("do something", mockHttpResponseException);
-
-    builder.addReason(
-        new ErrorEntryTemplate(ErrorCodes.MANIFEST_INVALID.name(), "manifest invalid"));
-    builder.addReason(new ErrorEntryTemplate(ErrorCodes.BLOB_UNKNOWN.name(), "blob unknown"));
-    builder.addReason(
-        new ErrorEntryTemplate(ErrorCodes.MANIFEST_UNKNOWN.name(), "manifest unknown"));
-    builder.addReason(new ErrorEntryTemplate(ErrorCodes.TAG_INVALID.name(), "tag invalid"));
-    builder.addReason(
-        new ErrorEntryTemplate(ErrorCodes.MANIFEST_UNVERIFIED.name(), "manifest unverified"));
-    builder.addReason(
-        new ErrorEntryTemplate(ErrorCodes.UNSUPPORTED.name(), "some other error happened"));
-    builder.addReason(new ErrorEntryTemplate("unknown", "some unknown error happened"));
+        new RegistryErrorExceptionBuilder("do something", mockHttpResponseException)
+            .addReason(
+                new ErrorEntryTemplate(ErrorCodes.MANIFEST_INVALID.name(), "manifest invalid"))
```

Comment:
Yeah, and this is what the TODO was saying. `ErrorEntryTemplate` is meant for representing a JSON. Here, the code is just re-purposing an existing JSON template only to be able to pass a code and a string conveniently. No JSON needs to be constructed really.

But I think for this cleanup, we are OK leave these as-is, as long as we leave the TODO I just mentioned.

Reply:
Makes sense.